### PR TITLE
Feature/jwt

### PR DESCRIPTION
**Pull request by ReamFOX**

### CODE REVIEW

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Please resolve all conflicts first

File: src/main/java/com/farion/onlinebookstore/security/impl/JwtAuthFilter.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@RequiredArgsConstructor
+@Component
+public class JwtAuthFilter extends OncePerRequestFilter {
+    private static final String HEADER = "Authorization";
```
Comment (yevheniisynytsia) [inline src/main/java/com/farion/onlinebookstore/security/impl/JwtAuthFilter.java]:
```suggestion
    private static final String AUTHORIZATION_HEADER_NAME = "Authorization";
```

File: src/main/java/com/farion/onlinebookstore/security/impl/JwtAuthFilter.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    private static final String TYPE = "Bearer ";
```
Comment (yevheniisynytsia) [inline src/main/java/com/farion/onlinebookstore/security/impl/JwtAuthFilter.java]:
```suggestion
    private static final String PREFIX_BEARER = "Bearer ";
```

**Review by yevheniisynytsia — APPROVED** [review]
Well done! Left some naming comments. Also it contains changes from your previous PR that are not related to this PR. Please try to avoid such situations

**Pull request merged**